src/GridBackground/index.tsx

```diff
@@ -51,14 +51,20 @@ const GridBackground = memo<GridBackgroundProps>(
       [reverse, colorFront, strokeWidth],
     );
 
+    const [randomGroup, setRandomGroup] = useState(
+      random ? [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14] : undefined,
+    );
+    useEffect(() => {
+      setRandomGroup((randomGroup) => (randomGroup ? shuffle(randomGroup) : randomGroup));
```

Comment:
I think here is a missing:

```
useEffect(() => {
  if (!random) return;
  setRandomGroup((randomGroup) => (randomGroup ? shuffle(randomGroup) : randomGroup));
},[random])

```

Reply:
It's intentional. The randomGroup will be undefined if random is false, the effect will do nothing.

Reply:
why `random` is true , the randomGroup will be setted?

Reply:
Within the component, the logic is like this:

> If `randomGroup` is defined, it means random mode is on, we'll set random group to stable default value on initial render and will randomize it after hydration

Because `randomGroup` is used to determine if random mode is on, `random` prop is only used as determine if `randomGroup` should be defined or undefined on initial render.

Reply:
I think the `random` is also need to be used to control random mode in all renderer . check the playground here: https://ui.lobehub.com/components/grid-background

<img width="888" alt="image" src="https://github.com/lobehub/lobe-ui/assets/28616219/f915195a-0692-49ca-a4b9-44e341e51c5a">

Reply:
Ah, I misunderstood the code. Fixed!